docs/activedocs.md

```diff
@@ -0,0 +1,155 @@
+                                       connections otherwise considered
+                                       insecure
+    -v --version                       Prints the version of this command
+       --verbose                       Verbose mode
+```
+
+
+### Apply
+
+* Update existing activedocs. Create new one if it does not exist.
+* `activedocs_id_or_system_name` positional argument is an activedocs reference.
+   It can be either activedocs `id`, or activedocs `system_name`.
+   Toolbox will figure it out.
+* Update to `published=true` activedocs by `--publish` flag.
+* Update to `published=false` method by `--hide` flag.
+* *The `--openapi-spec` flag is mandatory when the specified activedocs
+   is applied the first time. Otherwise an error will be returned*
+* Several other options can be set. Check `usage`
+
+```shell
+NAME
+    apply - activedocs apply
+
+USAGE
+    3scale activedocs activedocs apply <remote>
+    <activedocs_id_or_system_name>
+
+DESCRIPTION
+    Create or update an ActiveDocs
+
+OPTIONS
+    -d --description=<value>           Specify the description of the
+                                       ActiveDocs
+       --hide                          Specify it to hide the ActiveDocs on
+                                       the Developer Portal
+    -i --service-id=<value>            Specify the Service ID associated to
+                                       the ActiveDocs
+       --openapi-spec=<value>          Specify the swagger spec. Can be a
+                                       file, an URL or '-' to read from
+                                       stdin. This option is mandatory when
+                                       applying the ActiveDoc for the first
+                                       time
+    -p --publish                       Specify it to publish the ActiveDocs
+                                       on the Developer Portal. Otherwise it
+                                       will be hidden
+    -s --name=<value>                  Specify the name of the ActiveDocs
+       --skip-swagger-validations      Specify it to skip validation of the
+                                       Swagger specification
+
+OPTIONS FOR ACTIVEDOCS
+    -c --config-file=<value>           3scale toolbox configuration file
+                                       (default:
+                                       $HOME/.3scalerc.yaml)
+    -h --help                          show help for this command
+    -k --insecure                      Proceed and operate even for server
+                                       connections otherwise considered
+                                       insecure
+    -v --version                       Prints the version of this command
+       --verbose                       Verbose mode
+```
+
+### List
+
+```shell
+NAME
+    list - activedocs list
+
+USAGE
+    3scale activedocs activedocs list <remote>
+
+DESCRIPTION
+    List all defined ActiveDocs
+
+OPTIONS FOR ACTIVEDOCS
```

Comment:
It could be useful `--service` to list only activedocs for a certain service. But let's users demand first